js/site-nav.js

```diff
@@ -4,35 +4,44 @@
 
 var $ = require('jquery');
 
+/** SiteNav module
+  * On mobile: Controls the visibility of the the hamburger menu and sublists
+  * On desktop: Controls the visibility of dropdown sublists on hover and focus
+  * @constructor
+  * @param {object} selector - CSS selector for the nav component
+  */
+
 function SiteNav(selector) {
   this.$body = $(selector);
   this.$toggle = this.$body.find('.js-nav-toggle');
+  this.$openSublist = null;
 
   this.assignAria();
 
+  /** Open and close the menu on mobile **/
```

Comment:
The `/** ... */` comment style is for jsdoc-formatted comments specifically. For unstructured comments like this, I think `// ...` or `/* ... */` works fine.